collections/search/index.php

```diff
@@ -425,6 +386,141 @@
+									</label>
+									<span class="assistive-text"><?php echo $LANG['SEPARATE_MULTIPLE_W_COMMA'] ?></span>
+								</div>
+							</div>
+							<div>
+								<div>
+									<input type='checkbox' name='typestatus' id='typestatus' value='1' data-chip="<?php echo $LANG['ONLY_TYPE_SPECIMENS'] ?>" />
+									<label for="typestatus"><?php echo $LANG['TYPE'] ?></label>
+								</div>
+								<div>
+									<input type="checkbox" name="hasimages" id="hasimages" value=1 data-chip="<?php echo $LANG['ONLY_WITH_IMAGES'] ?>">
+									<label for="hasimages"><?php echo $LANG['LIMIT_TO_SPECIMENS_W_IMAGES'] ?></label>
+								</div>
+								<div>
+									<input type="checkbox" name="hasgenetic" id="hasgenetic" value=1 data-chip="<?php echo $LANG['ONLY_WITH_GENETIC'] ?>">
+									<label for="hasgenetic"><?php echo $LANG['LIMIT_TO_SPECIMENS_W_GENETIC_DATA'] ?></label>
+								</div>
+								<div>
+									<input type='checkbox' name='hascoords' id='hascoords' value='1' data-chip="<?php echo $LANG['ONLY_WITH_COORDINATES'] ?>" />
+									<label for="hascoords"><?php echo $LANG['HAS_COORDS'] ?></label>
+								</div>
+								<div>
+									<input type='checkbox' name='includecult' id='includecult' value='1' data-chip="<?php echo $LANG['INCLUDE_CULTIVATED'] ?>" <?php echo $SHOULD_INCLUDE_CULTIVATED_AS_DEFAULT ? 'checked' : '' ?> />
+									<label for="includecult"><?php echo $LANG['INCLUDE_CULTIVATED'] ?></label>
+								</div>
+							</div>
+							<?php
+								if($matSampleTypeArr = $collManager->getMaterialSampleTypeArr()){
+								?>
+								<div class="select-container">
+									<label for="materialsampletype"><?= $LANG['MATERIAL_SAMPLE_TYPE'] ?></label>
+									<select name="materialsampletype" id="materialsampletype">
+										<option id="materialsampletype-none" data-chip="<?php echo $LANG['MATERIAL_SAMPLE'] . ': ---' ?>" value="">---------------</option>
+										<option id="materialsampletype-all-ms" data-chip="<?php echo $LANG['MATERIAL_SAMPLE'] . ': ' . $LANG['ALL_MATERIAL_SAMPLE'] ?>" value="all-ms"><?= $LANG['ALL_MATERIAL_SAMPLE'] ?></option>
+										<?php
+										foreach($matSampleTypeArr as $matSampeType){
+											echo '<option data-chip=' . $LANG['MATERIAL_SAMPLE'] . ': ' . $matSampeType . 'value="' . $matSampeType . '">' . $matSampeType . '</option>';
+										}
+										?>
+									</select>
+								</div>
+								<?php
+							}
+							?>
+						</div>
+					</div>
+				</section>
+
+				<!-- Traits -->
+				<?php
+					if(!empty($SEARCH_BY_TRAITS)) {
+						$attribSearch = new OccurrenceAttributeSearch();
+						$traitArr = $attribSearch->getTraitSearchArr($SEARCH_BY_TRAITS);
+						if($traitArr){
+				?>
+							<section>
+								<!-- Accordion selector -->
+								<input type="checkbox" id="trait" class="accordion-selector" />
+								<!-- Accordion header -->
+								<label for="trait" class="accordion-header"><?php echo $LANG['TRIAT_CRITERIA'] ?></label>
```

Comment:
Typo (although it's consistent in your lang files too, haha!)

Reply:
a490b5535